lib/game.rb

```diff
@@ -153,17 +49,20 @@ def player_turn
         puts "Your shot on #{human_input} was a HIT."
       else
         @cpu.board.cells[human_input].render == "X"
-        puts "Your shot on #{human_input} sunk my battleship."
+        if @cpu_cruiser.sunk? 
+          puts "Your shot on #{human_input} sunk my #{@cpu_cruiser.name}"
+        else
+          puts "Your shot on #{human_input} sunk my #{@cpu_submarine.name}"
+        end
```

Comment:
this string interpolation defining the type of ship is really great. im proud of us!